addons/crm_mail_client_extension/data/ir_actions_data.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+    <odoo>
+        <record id="open_lead_edit_mode" model="ir.actions.act_window">
```

Comment:
Where is this used exactly?

Reply:
This action is needed by the plugin to redirect the user to the newly created lead (created by the crm_lead_create method) in edit mode when they click on the create lead button, if we use the standard action the user will not be redirected to the form in edit mode which is not a desirable behavior.

Reply:
I would rather have a specific route in the controller.
`/crm_lead/open` maybe ?

That takes the ID of the record and that simply redirects to an existing action specifying the ID.
We try to avoid creating actions when possible.

Reply:
As discussed on discord, it's unfortunately not possible to redirect the user to the lead form in edit mode, without creating a new action.

The name of the action has been changed to "action_lead_edit_mode", it has been moved to the crm_lead_views.xml file.